request.go

```diff
@@ -323,6 +334,7 @@ func (conn *Connection) CallAsync(functionName string, args interface{}) *Future
 // Call17Async sends a call to registered Tarantool function and returns Future.
 // It uses request code for Tarantool 1.7, so future's result will not be converted
 // (though, keep in mind, result is always array)
+//nolint: errcheck
```

Comment:
godoc renders this comment to documentation, see 
![image](https://user-images.githubusercontent.com/1151557/164522659-2138db7c-7f5e-4c79-b074-07b07ade4d52.png)

Reply:
That seems to be a problem to use the comment `//nolint` for suppressing some warnings. AFAIU, there is no way to say godoc to ignore some special comments in code (See https://github.com/golang/go/issues/20925). I would propose to leave some suppressions in check.yml.

Reply:
Seems resolved, so I'll resolve the discussion.